tests/test_views.py

```diff
@@ -25,19 +27,29 @@ def setUpClass(cls):
             related_urls=[],
         )
 
+        cls.superuser = User.objects.create_superuser(
+            username="test", email="test@greedybear.com", password="test"
+        )
+
     def test_for_vaild_unregistered_ip(self):
         """Check for a valid IP that is unavaliable in DB"""
+        self.client = APIClient()
```

Comment:
this row is not necessary cause the `TestCase` class already does this for you. So you can remove these lines in all the tests